aten/src/ATen/native/RNN.cpp

```diff
@@ -504,6 +518,7 @@ std::tuple<Tensor, Tensor> NAME(                                               \
       const Tensor& _input, const Tensor& hx,                                  \
       TensorList _params, bool has_biases,                                     \
       int64_t num_layers, double dropout_p, bool train, bool bidirectional, bool batch_first) { \
+  check_device(_input, hx);                                                    \
```

Comment:
Nice! Could you also check tensors in `_params` as well?

Reply:
Sure, so I added checks for `_params`. Let me know if I should add more test for it.